src/lib/devConsole.js

```diff
@@ -821,6 +821,64 @@ const getMeshArtifact = async (organizationId, projectId, workspaceId, workspace
 	}
 };
 
+/**
+ * Gets the enabled features for the tenant.
+ *
+ * This request bypasses the Dev Console and is sent directly to the Schema Management Service.
+ * As a result, we provide the orgCode instead of orgId since Dev Console usually performs the translation.
+ * The near-term goal is to stop using Dev Console as a proxy for all routes.
+ * @param organizationCode
+ * @returns {Promise<Object>}
+ */
+const getTenantFeatures = async organizationCode => {
```

Comment:
I added this function to this file despite it sending the request directly to SMS, instead of using the Dev Console proxy.

As far as I saw it, I had 3 options:
1. Create a separate module for the SMS client. It would then depend on this module as I don't want to duplicate code.
2. Extract the common functionality from this file, put it in it's own module, and make two separate clients (SMS and Dev Console) which required it. This would impact more touch points within the plugin and involve more refactoring than I was prepared to do.
3. Add it directly to this file despite it not really going to the Dev Console. It was the simplest way implement, and fairly acceptable since we are planning a full migration of all the requests. (opted for this route)

Reply:
After further testing, SMS does in fact authorize against a valid organization. That is, the user must belong to the organization represented by the organizationCode in the request parameters, otherwise a 403 Unauthorized response code is returned. This is verified by SMS using the IMS token provided on the request.

The CLI _should_ only be operating within an authorized org for the user. However, this means we are also covered for any API requests originating from outside the CLI 🙂